feathr_project/setup.py

```diff
@@ -61,26 +61,28 @@
     include_package_data=True,
     # consider
     install_requires=[
-        "click<=8.1.3",
```

Comment:
I've noticed several bugs caused by unexpected version conflict before. There are some potential risk removing those version ping before an overall validation.

Reply:
I think for those packages that are non-essential, like pandas or click, where we are just using its basic functionalities, we should remove those restrictions to make it easier to install.

For some versions that caused issues or we know won't work, like protobuf, or azure dependencies, we should have version restriction.